src/ssl.c

```diff
@@ -4463,6 +4463,14 @@ int wolfSSL_Init(void)
             WOLFSSL_MSG("Bad wolfCrypt Init");
             return WC_INIT_E;
         }
+
+#ifdef OPENSSL_EXTRA
+        if (wolfSSL_RAND_seed("", 0) != WOLFSSL_SUCCESS) {
```

Comment:
Can we pass NULL here instead of an empty string ("")?

Reply:
Fixed